### DISCUSSION TITLE
Property graph converted using to_networkx function cannot be saved correctly to gexf file

### DISCUSSION THREAD

**Discussion by Jasmine-lxn** [pyg-team/pytorch_geometric#4579] [Q&A]
### I want to use to_networkx function to convert the data in torch to networkx (retain the attributes of nodes and edges in the graph data), and then save it as a gexf file, but it always prompts an error, what should I do?

![2](https://user-images.githubusercontent.com/56641033/166175989-dc6d6431-bcda-4404-ab0c-15975a6e076c.png)

![1](https://user-images.githubusercontent.com/56641033/166175534-a8fc8117-f3b2-4be8-ada8-23fb45f1fa66.png)

**Comment by Padarn** [comment]
You seem to have posted this twice? It would also usually be easier to help if you provide your code in a code block rather than a screenshot.

In this case the error seems to occur because `networkx` is assuming that when you have a list as the value object then it is specifying an edge and it's value (from https://github.com/networkx/networkx/blob/7e17ff9007d2e71c2ea2a6d340889c331689343c/networkx/readwrite/gexf.py#L465)
```
            if isinstance(v, list):
                # dynamic data
                for val, start, end in v:
                    val_type = type(val)
```
Maybe its a good idea to raise an issue on their github?

**Reply by Jasmine-lxn** [reply]
Thank you very much for your suggestion, I will try again.
